core/src/main/java/discord4j/core/DiscordClientBuilder.java

```diff
@@ -69,11 +67,11 @@
 import java.util.concurrent.Executors;
 
 /**
- * Builder suited for creating a {@link discord4j.core.DiscordClient}.
+ * Builder suited for creating a {@link DiscordClient}.
  * <p>
  * Allows creating a Discord client for a single shard with many configurable options. You can reuse this builder for
  * sharding as long as you set the {@link #setShardIndex(Integer)} and {@link #setShardCount(Integer)}
- * options, or {@link #setIdentifyOptions(discord4j.gateway.IdentifyOptions)}.
+ * options, or {@link #setIdentifyOptions(IdentifyOptions)}.
  */
 public final class DiscordClientBuilder {
```

Comment:
I feel at this point DiscordClientBuilder is becoming a god object. This particular review is not a hindrance to pushing this PR but more to comment in that we need to seriously reconsider this entire design at a later date.